src/file.ts

```diff
@@ -3687,17 +3672,10 @@ class File extends ServiceObject<File, FileMetadata> {
           } else {
             pipeline(data, writable, err => {
               if (err) {
-                // If data is not a valid PipelineSource, then pipeline will
-                // fail without destroying the writable stream. If data is a
-                // PipelineSource that yields invalid chunks (e.g. a stream in
-                // object mode or an iterable that does not yield Buffers or
-                // strings), then pipeline will destroy the writable stream.
-                if (!writable.destroyed) writable.destroy();
```

Comment:
This case doesn't happen - instead, Node throws immediately and this callback is never called.